v1-2/learn/how-to-use-java-interoperability.md

```diff
@@ -0,0 +1,939 @@
+---
+layout: ballerina-inner-page
+title: How to Use Java Interoperability
+permalink: /v1-2/learn/how-to-use-java-interoperability/
+---
+
+# How to call Java code from Ballerina
+
+## Introduction
+Ballerina offers a straightforward way to call the existing Java code from Ballerina and also provides a Java API to call Ballerina code from Java.  Although Ballerina is not designed to be a JVM language, the current implementation, which targets the JVM, aka jBallerina, provides Java interoperability by adhering to Ballerina language semantics. 
+
+### Ballerina bindings to Java code
+Your task is to write Ballerina code (Ballerina bindings) that lets you call into the corresponding Java API. 
+
+[TODO] Image 
```

Comment:
Are we adding the image?

Reply:
I removed the image for the moment. It is not a blocker for the release. Let's add it later.